tools/update-authors-json

```diff
@@ -93,6 +98,16 @@ def run_production() -> None:
                         'avatar': contrib.get('author').get('avatar_url'),
                         name: contrib.get('total'),
                     }
+                    if full_names.get(username):
+                        contrib_data["fullname"] = full_names[username]
+                    else:
+                        user_details = requests.get('https://api.github.com/users/'+username).json()
+                        user_full_name = user_details.get('name')
+                        if not user_full_name:
+                            user_full_name = username
+                        full_names[username] = user_full_name
+                        contrib_data["fullname"] = user_full_name
```

Comment:
Can you have it print summary for how many users it needed to fetch names, so we can track how many requests it needs?

Reply:
Right now the full names of each user is saved in `fixtures/authors_full_names.json`. Whenever a new contributor comes on the scene, it will use the requests once and save the name in `fixtures/authors_full_names.json`.